### PR TITLE
Add support for airline warning and error colors

### PR DESCRIPTION
**Pull request by axelitus**
I noticed there weren't colors defined for the errors and warnings of airline, so I've added those to the airline nord theme.

### CODE REVIEW

**Comment by axelitus** [comment]
Without warninng color:
image

With warning color:
image

File: autoload/airline/themes/nord.vim [hunk 1]
```diff
@@ -45,24 +45,44 @@ let s:nord6_term = "15"
 let s:NMain = [s:nord1_gui, s:nord8_gui, s:nord1_term, s:nord8_term]
 let s:NRight = [s:nord1_gui, s:nord9_gui, s:nord1_term, s:nord9_term]
 let s:NMiddle = [s:nord5_gui, s:nord3_gui, s:nord5_term, s:nord3_term]
+let s:NWarn = [s:nord3_gui, s:nord13_gui, s:nord3_term, s:nord13_term]
```
Comment (arcticicestudio) [inline autoload/airline/themes/nord.vim]:
```suggestion
 let s:NWarn = [s:nord1_gui, s:nord13_gui, s:nord1_term, s:nord13_term]
```

File: autoload/airline/themes/nord.vim [hunk 2]
```diff
@@ -45,24 +45,44 @@ let s:nord6_term = "15"
...
+let s:NError = [s:nord0_gui, s:nord11_gui, s:nord1_term, s:nord11_term]
 let g:airline#themes#nord#palette.normal = airline#themes#generate_color_map(s:NMain, s:NRight, s:NMiddle)
+let g:airline#themes#nord#palette.normal.airline_warning = s:NWarn
+let g:airline#themes#nord#palette.normal.airline_error = s:NError
 
 let s:IMain = [s:nord1_gui, s:nord14_gui, s:nord1_term, s:nord6_term]
 let s:IRight = [s:nord1_gui, s:nord9_gui, s:nord1_term, s:nord9_term]
 let s:IMiddle = [s:nord5_gui, s:nord3_gui, s:nord5_term, s:nord3_term]
+let s:IWarn = [s:nord3_gui, s:nord13_gui, s:nord3_term, s:nord13_term]
```
Comment (arcticicestudio) [inline autoload/airline/themes/nord.vim]:
```suggestion
let s:IWarn = [s:nord1_gui, s:nord13_gui, s:nord1_term, s:nord13_term]
```

File: autoload/airline/themes/nord.vim [hunk 3]
```diff
@@ -45,24 +45,44 @@ let s:nord6_term = "15"
...
+let s:IError = [s:nord0_gui, s:nord11_gui, s:nord1_term, s:nord11_term]
 let g:airline#themes#nord#palette.insert = airline#themes#generate_color_map(s:IMain, s:IRight, s:IMiddle)
+let g:airline#themes#nord#palette.insert.airline_warning = s:IWarn
+let g:airline#themes#nord#palette.insert.airline_error = s:IError
 
 let s:RMain = [s:nord1_gui, s:nord14_gui, s:nord1_term, s:nord14_term]
 let s:RRight = [s:nord1_gui, s:nord9_gui, s:nord1_term, s:nord9_term]
 let s:RMiddle = [s:nord5_gui, s:nord3_gui, s:nord5_term, s:nord3_term]
+let s:RWarn = [s:nord3_gui, s:nord13_gui, s:nord3_term, s:nord13_term]
```
Comment (arcticicestudio) [inline autoload/airline/themes/nord.vim]:
```suggestion
let s:RWarn = [s:nord1_gui, s:nord13_gui, s:nord1_term, s:nord13_term]
```

File: autoload/airline/themes/nord.vim [hunk 4]
```diff
@@ -45,24 +45,44 @@ let s:nord6_term = "15"
...
+let s:RError = [s:nord0_gui, s:nord11_gui, s:nord1_term, s:nord11_term]
 let g:airline#themes#nord#palette.replace = airline#themes#generate_color_map(s:RMain, s:RRight, s:RMiddle)
+let g:airline#themes#nord#palette.replace.airline_warning = s:RWarn
+let g:airline#themes#nord#palette.replace.airline_error = s:RError
 
 let s:VMain = [s:nord1_gui, s:nord7_gui, s:nord1_term, s:nord7_term]
 let s:VRight = [s:nord1_gui, s:nord9_gui, s:nord1_term, s:nord9_term]
 let s:VMiddle = [s:nord5_gui, s:nord3_gui, s:nord5_term, s:nord3_term]
+let s:VWarn = [s:nord3_gui, s:nord13_gui, s:nord3_term, s:nord13_term]
```
Comment (arcticicestudio) [inline autoload/airline/themes/nord.vim]:
```suggestion
let s:VWarn = [s:nord1_gui, s:nord13_gui, s:nord1_term, s:nord13_term]
```

File: autoload/airline/themes/nord.vim [hunk 5]
```diff
@@ -45,24 +45,44 @@ let s:nord6_term = "15"
...
+let s:VError = [s:nord0_gui, s:nord11_gui, s:nord1_term, s:nord11_term]
 let g:airline#themes#nord#palette.visual = airline#themes#generate_color_map(s:VMain, s:VRight, s:VMiddle)
+let g:airline#themes#nord#palette.visual.airline_warning = s:VWarn
+let g:airline#themes#nord#palette.visual.airline_error = s:VError
 
 let s:IAMain = [s:nord5_gui, s:nord3_gui, s:nord5_term, s:nord3_term]
 let s:IARight = [s:nord5_gui, s:nord3_gui, s:nord5_term, s:nord3_term]
 let s:IAMiddle = [s:nord5_gui, s:nord1_gui, s:nord5_term, s:nord1_term]
+let s:IAWarn = [s:nord3_gui, s:nord13_gui, s:nord3_term, s:nord13_term]
```
Comment (arcticicestudio) [inline autoload/airline/themes/nord.vim]:
```suggestion
let s:IAWarn = [s:nord1_gui, s:nord13_gui, s:nord1_term, s:nord13_term]
```

**Comment by axelitus** [comment]
No worries, I know how too busy can feel overwhelming sometimes. I'll take a look at it later today or tomorrow.

**Comment by axelitus** [comment]
Totally agreed with your comments. It's better with more contrast.

I have added a commit which implements your suggestions.

**Comment by arcticicestudio** [comment]
I've rebased your branch on top of the `develop` branch and resolved the merge conflict and will merge it to deploy it with the next release version :rocket: 
Thanks again for the PR :+1:

**Pull request merged**